pkg/sql/codegen/couler/codegen_test.go

```diff
@@ -115,3 +117,59 @@ func TestWriteArgoYamlWithClusterConfig(t *testing.T) {
 
 	a.Equal(string(out), expectedArgoYAML)
 }
+
+func TestKatibCodegen(t *testing.T) {
+	a := assert.New(t)
+	os.Setenv("SQLFLOW_submitter", "katib")
+
+	cfg := &mysql.Config{
```

Comment:
Can using [testingMySQLURL](https://github.com/sql-machine-learning/sqlflow/blob/develop/pkg/database/testing.go#L85) instead of a hardcode `mysql.Config`